src/patch-global.js

```diff
@@ -0,0 +1,7 @@
+if (!globalThis.URLPattern) {
+  const { URLPattern } = await import("./index.raw.js");
+  globalThis.URLPattern = URLPattern;
+}
+
+const URLPattern = globalThis.URLPattern;
```

Comment:
I don't think this needs an export - it can be side-effect only. Users can import the implementation directly if they need to.

Reply:
OK removed